docs/Unicode.md

```diff
@@ -52,7 +86,9 @@ From the error in the image above, Thai characters can be fixed using fonts from
 
 ![thai-font-working](https://raw.githubusercontent.com/py-pdf/fpdf2/master/tutorial/thai-accent-working.png)
 
```

Comment:
This Thai font problem doesn't happen when text shaping is enabled

Reply:
So most of this section (except the first paragraph) can go?

Reply:
Correct